client/public/index.html

```diff
@@ -60,24 +60,45 @@
         });
       }
 
-      // 4. The API will call this function when the video player is ready.
       function onPlayerReady(event) {
+        console.debug("onPlayerRead", event);
         event.target.playVideo();
       }
 
-      // 5. The API calls this function when the player's state changes.
-      //    The function indicates that when playing a video (state=1),
-      //    the player should play for six seconds and then stop.
-      var done = false;
       function onPlayerStateChange(event) {
-        if (event.data == YT.PlayerState.PLAYING && !done) {
-          setTimeout(stopVideo, 6000);
-          done = true;
-        }
+        console.debug("onPlayerStateChange", event);
       }
+
+      function playVideo() {
```

Comment:
this seemed to be the easiest way to hook into the player's functions. just put it all on the window #YOLO